<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDialog</class>
 <widget class="QDialog" name="SetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="trackCombo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Straight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Curve</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Complex</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Track</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>number of cars</string>
   </property>
  </widget>
  <widget class="QComboBox" name="numCombo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Done!</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>189</y>
     <width>691</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="inputLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit1"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label3">
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label2">
      <property name="text">
       <string>name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit2"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit3"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>speed</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>speed</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>speed</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
